Storage/SQL/SQLitePasswords.swift

```diff
@@ -34,9 +38,20 @@ private class PasswordsTable<T>: GenericTable<Password> {
         args.append(item.timeCreated)
         args.append(item.timeLastUsed)
         args.append(item.timePasswordChanged)
+
         args.append(item.username)
-        args.append(item.password)
-        return ("INSERT INTO \(name) (hostname, httpRealm, formSubmitUrl, usernameField, passwordField, guid, timeCreated, timeLastUsed, timePasswordChanged, username, password) VALUES (?,?,?,?,?,?,?,?,?,?,?)", args)
+        switch (encryption) {
+        case EncryptionType.AES256:
+            let secret = SQLitePasswords.secretKey!
+            args.append(item.password.AES256EncryptWithKey(secret))
+            args.append(EncryptionType.AES256.rawValue)
+        default:
+            args.append(item.password)
```

Comment:
Rather than 6 line notes about this, a single one with "I don't think we should offer a no-encryption option" would probably do the same thing. I don't have strong opinions. I just want to make sure the type/version of encryption is tracked so that we can change it as we need to.